static/js/src/smartquotes.js

```diff
@@ -0,0 +1,3 @@
+var smartquotes = require("smartquotes");
+
+smartquotes(document.getElementById("main-content"));
```

Comment:
Only focusing the `main-content` means there could be dumb quotes in the navigation and the footer still. Do we take a big performance hit if we apply it to the whole page?